### PR TITLE
Add initial reader

### PR DESCRIPTION
**Pull request by drrzmr**

### CODE REVIEW

File: docker/base-images/golang:1.9.2-stretch/Dockerfile [hunk 1]
```diff
@@ -10,3 +10,16 @@ RUN set -eux; \
 		${GOPATH}/src/* \
 		${GOPATH}/pkg/*; \
 	strip ${GOPATH}/bin/*
+
```
Comment (jonatasnona) [inline docker/base-images/golang:1.9.2-stretch/Dockerfile]: Hi @drrzmr 
could you please describe more to us why you decide to add this layer in our golang based docker image?

thanks in advance.

File: docker/test/reader/run.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -eux
+
+docker-compose up -d
+docker-compose exec golang bash -c 'psql < /scripts/sql/reader/start.sql'
+docker-compose exec golang bash -c 'psql < /scripts/sql/reader/events.sql'
+docker-compose exec golang bash -c 'pg_recvlogical -d reader -h postgres-server -S td -v --start -f - | go run main.go reader' || \
```
Comment (drrzmr) [inline docker/test/reader/run.sh]: @jonatasnona Here is the answer for your question about postgres-client layer, that layer is needed to put pg_recvlogical on golang image and we build something like this line.
Golang image/container is just for build/development/test purposes, so I guess that a extra layer are not bad and make our life simplier.
For production we can use postgres-client image, using lighter approach.

Comment (jonatasnona) [reply docker/test/reader/run.sh]: now i see, thanks bro to clarify.

File: reader/reader_test.go [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+	// buf -> sw -> sr
+	go func() {
+		buf := []byte(text)
+		n, err := sw.Write(buf)
+		require.Equal(t, len(buf), n)
+		require.NoError(t, err)
+
+		err = sw.Close()
+		require.NoError(t, err)
+
```
Comment (jonatasnona) [inline reader/reader_test.go]: remove unnecessary space.

**Pull request merged**